UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -790,6 +790,29 @@ public override bool Equals(object obj)
             return AsBaseUnitMetersPerSecondSquared().Equals(((Acceleration) obj).AsBaseUnitMetersPerSecondSquared());
         }
 
+        /// <summary>
+        ///     Compare equality to another Acceleration within the given absolute or relative tolerance.
+        ///     Relative tolerance is when the difference between the two quantities is not greater than the scale of the values compared to the tolerance.
```

Comment:
How about:
> Relative tolerance is defined by a fraction of this quantity, so a value of 0.01 means +/- 1% tolerance.
> Absolute tolerance is defined by the max difference between this quantity and <paramref name="other" /> in the <see cref="Unit" /> of this quantity. So if this quantity was constructed with 100 cm, then a tolerance of 0.01 means a tolerance of +/- 1 cm.

When writing this, I realize that I'm not sure this behavior for absolute tolerance is intuitive enough and whether we should have a separate method for absolute equality that instead take an `Acceleration` as tolerance? The relative tolerance implementation is fine I think.

Reply:
I think having an example is good here. Makes it easier to understand. I'll add that. 

As for making the method typed I'm not so sure. When we want to implement an IQuantity interface, having the generic method that takes a double is the only way to make it reusable as an interface method.

Reply:
That is true with the interface, but generally I'd expect a check, much like Equals(Object) does. If the types are different it would just always return false. But all debatable :)

Are you saying you'd like to see:

```csharp
var a = Length.FromMeters( 5.0 );
var isEqual = a.EqualsAbsolute( Length.FromMeters( 10.0 ), Length.FromNauticalMiles( 100000 ) )
```